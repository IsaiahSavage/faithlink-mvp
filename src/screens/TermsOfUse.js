import { StyleSheet, Text, View } from 'react-native';
import React from 'react';

const TermsOfUse = () => {
  return (
    <View style={styles.container}>
      <Text style={styles.title}>Terms of Use</Text>
      <Text style={styles.text}>Coming soon.</Text>
    </View>
  );
};

export default TermsOfUse;

const styles = StyleSheet.create({
  container: {
    flex: 1,
    alignItems: 'center',
    justifyContent: 'center',
  },
  title: {
    fontSize: 25,
  },
});
