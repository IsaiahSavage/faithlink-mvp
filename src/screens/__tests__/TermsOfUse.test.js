import React from 'react';
import { render } from '@testing-library/react-native';
import TermsOfUse from '../TermsOfUse'

describe('PrivacyPolicy', () =>{
    //see if it renders
    it('page renders correectly', () =>{
       render(<TermsOfUse/>); 
    });

    //is not changing between loads
    it('matches the snapshot', () => {
        const { toJSON } = render(<TermsOfUse />);
        expect(toJSON()).toMatchSnapshot();
      });

    //text matches
    it('text is the same', () =>{
        const {getByText} = render(<TermsOfUse/>);
        const title = getByText(/Terms of Use/);
        const text = getByText(/Coming soon./);
        expect(title).toBeTruthy();
        expect(text).toBeTruthy();
    });
});