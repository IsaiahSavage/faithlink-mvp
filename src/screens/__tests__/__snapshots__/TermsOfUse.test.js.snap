// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrivacyPolicy matches the snapshot 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      {
        "fontSize": 25,
      }
    }
  >
    Terms of Use
  </Text>
  <Text>
    Coming soon.
  </Text>
</View>
`;
